Doc/library/mmap.rst

```diff
@@ -94,8 +94,8 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
 
    *offset* may be specified as a non-negative integer offset. mmap references
    will be relative to the offset from the beginning of the file. *offset*
-   defaults to 0.  *offset* must be a multiple of the PAGESIZE or
-   ALLOCATIONGRANULARITY.
+   defaults to 0.  *offset* must be a multiple of the :const:`PAGESIZE` or
+   :const:`ALLOCATIONGRANULARITY`.
```

Comment:
`ALLOCATIONGRANULARITY` is equivalent of `PAGESIZE` under Unix. Since this documentation for the Unix version of `mmap.mmap()`, perhapd we can remove `ALLOCATIONGRANULARITY` from the documentation?

Reply:
Nit: "a multiple of the"

Can "the" be removed here?

Reply:
I think it is better to use the same condition on all platforms. Since ALLOCATIONGRANULARITY is equivalent of PAGESIZE under Unix, you can use ALLOCATIONGRANULARITY as well as PAGESIZE.

Maybe change the wording to something like: "offset must be a multiple of ALLOCATIONGRANULARITY which is equal to PAGESIZE under Unix"?